client/components/skills/skills.js

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect} from 'react'
+import {useSelector, useDispatch} from 'react-redux'
+import {fetchAllSkills} from '../../store/skill'
+import {Link} from 'react-router-dom'
+
+export default function Skills() {
+  const dispatch = useDispatch()
+  const allSkills = useSelector(state => state.skills)
+
+  useEffect(() => {
+    dispatch(fetchAllSkills())
+  }, [])
+
+  return (
+    <>
+      {allSkills && allSkills.length ? (
+        allSkills.map(skill => (
+          <div key={skill.id}>
+            <Link to={`/skills/${skill.id}`}>{skill.title}</Link>
```

Comment:
What's the idea behind having a route to 'skills/:id' ? are we gonna display information about that skill on a singular page? Perhaps list all the people who share that skill?